test/constraints/constraint_unit.jl

```diff
@@ -1428,6 +1428,31 @@ function test_constraint_ramp_up()
             observed_con = constraint_object(constraint[con_key...])
             @test _is_constraint_equal(observed_con, expected_con)
         end
+
+        max_startup_ramp = 0.4
```

Comment:
Is max_startup_ramp actually used in this test? Seems like the constraint function actually calls ramp_up_limit. Could we have tests that cover the two cases 1) max_startup_ramp defined/not defined?

Reply:
Actually, I guess you are testing the "is defined" case here, and probably the not-defined case has been added beforehand, so should be good

Reply:
Thanks for the review! Exactly, this block tests the case where `max_startup_ramp` is defined. And the previous block tests the case where it is not defined with the default value `None`.